### PR TITLE
Add wifi settings endpoint back

### PR DESCRIPTION
**Pull request by jkilpatr**
I should not have deleted GET wifi settings, which is actively used
along with POST wifi settings which was removed and no longer used.

### CODE REVIEW

File: rita/src/rita_client/dashboard/wifi.rs [hunk 1]
```diff
@@ -93,3 +96,51 @@ impl Handler<WifiPass> for Dashboard {
...
     }
 }
+
+pub struct GetWifiConfig;
+
+impl Message for GetWifiConfig {
+    type Result = Result<Vec<WifiInterface>, Error>;
+}
+
+impl Handler<GetWifiConfig> for Dashboard {
```
Comment (drozdziak1) [inline rita/src/rita_client/dashboard/wifi.rs]: Why exactly does this need to happen in the Dashboard actor? This message would generally clutter `Dashboard`'s mailbox while the data could be perfectly obtained from the request handler

File: rita/src/rita_client/dashboard/network_endpoints.rs [hunk 2]
```diff
@@ -256,3 +256,14 @@ fn validate_config_value(s: &str) -> Result<(), ValidationError> {
         Ok(())
     }
 }
+
+pub fn get_wifi_config(
```
Comment (drozdziak1) [inline rita/src/rita_client/dashboard/network_endpoints.rs]: [OCD Tier] You're calling the endpoint `/wifi_settings`, why is it `[...]_wifi_config` all of the sudden? Consistency helps keep things organized, having redundant names for the same (or closely coupled) thing is avoidably adding to the complexity of our code.

File: rita/src/rita_client/dashboard/wifi.rs (hunk 1 above)
Comment (jkilpatr) [reply rita/src/rita_client/dashboard/wifi.rs]: it's this or we put everything into the network endpoints receiver functions.  Might be better, but not by much.

File: rita/src/rita_client/dashboard/network_endpoints.rs (hunk 2 above)
Comment (jkilpatr) [reply rita/src/rita_client/dashboard/network_endpoints.rs]: this is how it was named before I ever touched it. Not sure it's worth trying to fix right now.

**Pull request merged**